_docs_integrate/data-model-overview.md

```diff
@@ -869,15 +851,14 @@ Theoretically you can send any kind of data in a RelationshipTemplate. However,
 | onNewRelationship      | [`Request`](#request)                | The Request that should pop up to the user in case there is no Relationship yet. In this Request you can send Attributes of yourself the user needs to in order to know who's RelationshipTemplate it is (e.g. company name, address, ...), as ask for Attributes of the user you need to know in the Relationship, or send some information you already know about the user, so it can be saved in its wallet (e.g. the customer id). |
 | onExistingRelationship | [`Request`](#request) \| `undefined` | The Request that should pop up to the user in case a Relationship already exists. An example usage is a Request with an `AuthenticationRequestItem` for the sake of two-factor authentication.                                                                                                                                                                                                                                         |
 
-## RelationshipCreationChangeRequestContent
+## RelationshipCreationContent
 
-The naming on this one in combination with its `response` property is a bit confusing. Even though the `RelationshipCreationChangeRequestContent` contains the word "Request", it has a `response` property.
-This is because in the context of Relationships, there are [RelationshipChanges](#relationshipchange), which have a `request` and a `response` property. But caution: these have nothing to do with the Content-types `Request` and `Response`.
+Theoretically you can send any kind of data in a Relationship's creation content. However, if the RelationshipTemplate's content was of type `RelationshipTemplateContent`, the creation content must be of type `RelationshipCreationContent`, containing the Response to the Template's Request on new Relationship.
```

Comment:
I would write `creationContent`  and `content`  and in the last sentence there must be ... on **a**  new Relationship.